### ISSUE TITLE
[Enhancement] helper - Uninstall-ChocolateyPath

### ISSUE THREAD

**Issue by rismoney** [chocolatey/chocolatey#97]
we probably need a whole bunch of helpers like $subject, but this one seems like a fun one to start on.  seems like fun in the semi-colon sense ;)

Could probably refactor Install to make its counterpart I'm guessing.

**Comment by khellang** [comment]
We'd :heart: this for [scriptcs](https://github.com/scriptcs/scriptcs) :+1:

**Comment by gep13** [comment]
@ferventcoder have you reviewed https://github.com/chocolatey/chocolatey/pull/542?  Seems like something we could include, but we would need some tests around it.

**Comment by ferventcoder** [comment]
Yeah. A whole bunch of helpers for uninstall would be good. Let's see how much choco itself can do without helpers in a parallel track. I'd like to pick this back up once the posh runner uses runspaces.
